scripts/actions/__tests__/serialize-mdx.test.js

```diff
@@ -312,3 +312,13 @@ test("test <InlineCode> element serializes and adds 'notranslate' class to eleme
 
   expect(html).toMatchSnapshot();
 });
+
+test("test <code> element serializes and adds 'notranslate' class to element", async () => {
+  const mdx = `
+<code>agent.report_custom_element</code>
```

Comment:
doesn't the MDX string here need to look like:
```
`agent.report_custom_element`
```

Reply:
we have instances of `<code>thing</code>` in MDX files, so i think we need both, i'll add a test

Reply:
i only see one MDX file in `src/content/docs` that uses `<code>`. Backticks are def the most common way to add inline code. 

And good call to add a test.

Reply:
right right right, i was looking at a snapshot turns out